code/datums/jobs.dm

```diff
@@ -599,15 +599,16 @@
 	limit = 5
 	wages = PAY_DOCTORATE
 	slot_back = /obj/item/storage/backpack/medic
-	slot_belt = /obj/item/device/pda2/medical
+	slot_belt = /obj/item/storage/belt/medical
 	slot_jump = /obj/item/clothing/under/rank/medical
 	slot_suit = /obj/item/clothing/suit/labcoat
 	slot_foot = /obj/item/clothing/shoes/red
 	slot_lhan = /obj/item/storage/firstaid/regular/doctor_spawn
 	slot_ears = /obj/item/device/radio/headset/medical
 	slot_eyes = /obj/item/clothing/glasses/healthgoggles
-	slot_poc1 = /obj/item/paper/book/pocketguide/medical
-	items_in_backpack = list(/obj/item/crowbar, /obj/item/robodefibrillator) // cogwerks: giving medics a guaranteed air tank, stealing it from roboticists (those fucks)
+	slot_poc1 = /obj/item/device/pda2/medical
+	slot_poc2 = /obj/item/storage/medical_pouch
```

Comment:
Hard disagree on adding the pouch. These are intended for Nuke Ops "Combat Medics", not regular doctors.